test/reinterpretarray.jl

```diff
@@ -160,3 +160,14 @@ let a = [0.1 0.2; 0.3 0.4], at = reshape([(i,i+1) for i = 1:2:8], 2, 2)
     r = reinterpret(Int, vt)
     @test r == OffsetArray(reshape(1:8, 2, 2, 2), (0, offsetvt...))
 end
+
+@testset "potentially aliased copies" begin
+    buffer = UInt8[1,0,0,0,0,0,0,0,2,0,0,0,0,0,0,0]
+    mid = length(buffer) ÷ 2
+    x1 = reinterpret(Int, @view buffer[1:mid])
```

Comment:
Better use `Int64`, I think this will fail on 32-bit.

Reply:
Indeed, from AV 32-bit:
```
Error in testset reinterpretarray:
Test Failed at C:\projects\julia\julia-\share\julia\test\reinterpretarray.jl:170
  Expression: x1 == x2 == [2]
   Evaluated: [2, 0] == [2, 0] == [2]
```

Reply:
Of course, thank you!